src/ChineseNumbers/StringUtils.h

```diff
@@ -0,0 +1,12 @@
+#ifndef FCITX5_MCBOPOMOFO_STRINGUTILS_H
+#define FCITX5_MCBOPOMOFO_STRINGUTILS_H
+
+#include <string>
+
+namespace ChineseNumbers::StringUtils {
```

Comment:
Here though using a nested namespace makes sense. I would prefer the truly-nested declaration though:

```cpp
namespace ChineseNumbers {

namespace StringUtils {

// ...

}  // namespace StringUtils

}  // namespace ChineseNumbers
```

Reply:
Fixed in 92e46601d055fb57699128d5d2cb8634a1fef339